tests/check_merge_conflict_test.py

```diff
@@ -128,5 +128,12 @@ def test_ignores_binary_files():
 
 
 def test_does_not_care_when_not_in_a_merge(tmpdir):
-    tmpdir.join('README.md').write('problem\n=======\n')
-    assert detect_merge_conflict(['README.md']) == 0
+    f = tmpdir.join('README.md')
+    f.write('problem\n=======\n')
+    assert detect_merge_conflict([str(f.realpath())]) == 0
```

Comment:
can just use `f.strpath` here instead of `str(f.realpath())`

thanks for fixing this test too 👍

Reply:
`strpath` is not officially [documented](http://py.readthedocs.io/en/latest/path.html#py._path.local.LocalPath). this is why I've used `realpath`

Reply:
interesting! `str(f)` will work too.  I'll work on getting that documented from the `py` side 😆